### ISSUE TITLE
Browser validator issue for ieeg mef3 file.

### ISSUE THREAD

**Issue by dorahermes** [bids-standard/legacy-validator#2089]
When I use the bids-validator in the [browser](https://bids-standard.github.io/bids-validator/), I get an error that there is an extension mismatch for my ieeg mef3 file:

Extension used by file does not match allowed extensions for its suffix:

```
/sub-ieegModulator/ses-ieeg01/ieeg/sub-ieegModulator_ses-ieeg01_task-photicstim_run-01_ieeg.mefd/RO13.timd/RO13-000001.segd/RO13-000001.tdat
```

This issue is present because the mef3 file is actually a .mefd folder with multiple folders/files for all channels. 

This issue is not present when submitting data to OpenNeuro, and that version of the validator correctly detects the .mefd folder as the ieeg mef3 data file.

**Comment by effigies** [comment]
Is there a dataset on OpenNeuro that has this issue? I'd like to try to reproduce it.

**Comment by effigies** [comment]
I was able to reproduce with https://openneuro.org/datasets/ds003708:

![image](https://github.com/user-attachments/assets/7381de0e-ad54-48fd-9511-fce5c7c9a4ee)

This only happened when I `datalad unlock`ed it, so there are a couple things happening here. Will look more into it.

**Comment by effigies** [comment]
Good news: It works on the development version. So should be resolved in the next release.

![image](https://github.com/user-attachments/assets/39eb6db1-c1eb-44cd-9768-079f7cc8f161)

Probably fixed by #2064, which I didn't realize didn't make it into last week's release.
